cpp/benchmarks/common/generate_input.cu

```diff
@@ -549,7 +549,7 @@ struct create_rand_col_fn {
                                            thrust::minstd_rand& engine,
                                            cudf::size_type num_rows)
   {
-    if (profile.get_cardinality() == 0 || profile.get_cardinality() >= num_rows) {
+    if (profile.get_cardinality() >= num_rows) {
```

Comment:
Should there be a check for negative cardinality here?

Reply:
What should be the behavior for negative cardinality?

Reply:
>What should be the behavior for negative cardinality?

It should be an error I believe?